tests/SpecTests/RetryableWritesSpecTest.php

```diff
@@ -107,7 +107,7 @@ public function testNoWritesPerformedErrorReturnsOriginalError(): void
         ]);
 
         $subscriber = new class ($this) implements CommandSubscriber {
-            private $testCase;
+            private FunctionalTestCase $testCase;
```

Comment:
Since this is only used within this class, would it make sense to use `self`? This was conveniently [added in PHP 7.4](https://php.watch/versions/7.4/typed-properties).

FunctionalTestCase here refers to SpecTests/FunctionalTestCase, even though this actually only requires the base FunctionalTestCase class.

Reply:
Since we're in a new class definition, `self` won't do what we expect it to do, as it will refer to the anonymous class: https://3v4l.org/GavRg

Reply:
That's a good point. I've updated it to `RetryableWritesSpecTest`.